devhelp/src/dhp-manpages.c

```diff
@@ -0,0 +1,240 @@
+#include <stdlib.h>
```

Comment:
Could you add the file header with the authors and license in all source files?

Reply:
Yes, done.